internal/checker/terraform/rules/aws/aws_backup_selection_ebs_added_in_backup_plans_of_aws_backup.rego

```diff
@@ -0,0 +1,64 @@
+# METADATA
+# title: "Make certain that AWS Backup includes Elastic Block Store (EBS) volumes within its backup plans"
+# description: "Integrate EBS volumes into your AWS Backup strategies. Through the formulation of a backup plan, you can establish a routine for protecting your data, facilitating restoration should data loss or corruption arise."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/backup_selection.html
+# custom:
+#   id: CB_TFAWS_338
+#   severity: LOW
+package lib.terraform.CB_TFAWS_338
+
+isvalid(block){
+	block.Type == "resource"
+    block.Labels[_] == "aws_backup_selection"
+}
+
+resource[resource] {
+    block := pass[_]
+	resource := concat(".", block.Labels)
+} 
+
+resource[resource] { 
+    block := fail[_]
+	resource := concat(".", block.Labels)
+} 
+
+getTheLabelforAwsEBS_volume[label]{
+    resource := input[_]
+    resource.Type == "resource"
+    resource.Labels[_] == "aws_ebs_volume"
+    label := concat(".", resource.Labels)
+}
+
+isValidResourceAttached{
+    resource := input[_]
+    resource.Type == "resource"
+    resource.Labels[_] == "aws_backup_selection"
+    resources := resource.Attributes.resources[_]
```

Comment:
Kindly do not merge, this should not be here I will remove it after my current task.

Reply:
noted